doc/snek.adoc

```diff
@@ -2081,14 +2125,14 @@ return -1 if no characters are pending.(((stdscr.getch)))
 
 == Snek Development Environment
 
-The Snek Development Environment, snekde, is a Python program which runs
+The Snek Development Environment, Snekde, is a Python program which runs
 on Linux, Mac OS X and Windows to work with small devices running
 Snek, such as the Duemilanove and Metro M0 Express boards.(((snekde)))
 
 === Starting Snekde
 
-On Windows and Linux, launch snekde from your application menu. On Mac OS
-X, snekde is installed along with the other snek files in the Snek
+On Windows and Linux, launch `snekde` from your application menu. On Mac OS
+X, Snekde is installed along with the other Snek files in the Snek
```

Comment:
`snekde` and Snekde in the same paragraph? Should those be the same?

Reply:
Snekde is a component of Snek. I think it's fine.

Reply:
no, using `snekde` one place and Snekde in the other? Is that good?